cypress/integration/e2e/header-control-menu/menu-plik.spec.ts

```diff
@@ -1,22 +1,19 @@
-/// <reference types="cypress" />
-
 context("Menu 'plik'", () => {
   beforeEach(() => {
     cy.visit(Cypress.env("baseUrl"));
   });
 
   describe("File button", () => {
     it("Should open options menu", () => {
-      cy.contains("Plik").click();
-      cy.contains("Wczytaj");
-      cy.contains("Zapisz jako...");
+      cy.get("[data-cy=file-dropdown]").click();
```

Comment:
Było to całkiem nieźle zrobione, ale można lepiej według [best practises](https://docs.cypress.io/guides/references/best-practices.html#How-It-Works:~:text=cy.get('%5Bdata%2Dcy%3Dsubmit%5D').click()%09%20Always).